### ISSUE TITLE
ENOENT error in 0.3.6

### ISSUE THREAD

**Issue by pdkovacs** [karma-runner/karma-junit-reporter#67]
v0.3.6 started to fail with in our environment:
```
Firefox 35.0.0 (Ubuntu 0.0.0): Executed 648 of 648 SUCCESS (3.967 secs / 3.86 secs)
 [33mWARN [reporter.junit]: [39mCannot write JUnit xml
	 ENOENT, open '/data/slave/npsci/workspace/xxxx/front-end/test-reports/Firefox_35.0.0_(Ubuntu_0.0.0)/test-results.xml'
```

**Comment by lsdafjklsd** [comment]
Ya, we're seeing the same issue in v0.3.6.

**Comment by redowl** [comment]
+1 - seeing the same issue with builds in a Bamboo environment

**Comment by af7** [comment]
+1, I also have the following message:
30 09 2015 16:30:14.399:WARN [reporter.junit]: Cannot write JUnit xml

Also, I used to write my XML file at `projectroot/coverage/test-results.xml` with an old version. Now I see that you add the `PhantomJS_1.9.8_(Windows_8_0.0.0)` folder too. Since we are using JENKINS and create the coverage from different machines, **how can I please NOT include this folder**? This is what I have in my `karma.conf` file:
````
        junitReporter: {
            outputFile: 'coverage/test-results.xml',
            suite: ''
        },
````

**Comment by adriaanthomas** [comment]
This is fixed in #70. Until a new version is released, you can use this in your `package.json`:

```json
    "karma-junit-reporter": "karma-runner/karma-junit-reporter#ab9b155",
```

**Comment by Dignifiedquire** [comment]
Fixed in the latest release
